packages/mdx-embed/.eslintrc.js

```diff
@@ -17,12 +17,23 @@ module.exports = {
     },
   },
   rules: {
-    'react/prop-types': 0,
-    '@typescript-eslint/no-explicit-any': 0,
-    '@typescript-eslint/ban-ts-comment': 0,
-    '@typescript-eslint/ban-types': 0,
-    '@typescript-eslint/explicit-module-boundary-types': 0,
-    '@typescript-eslint/no-empty-interface': 0,
-    '@typescript-eslint/no-non-null-assertion': 0,
+    'react/prop-types': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/ban-ts-comment': 'off',
+    '@typescript-eslint/ban-types': 'off',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/no-empty-interface': 'warn',
+    '@typescript-eslint/no-non-null-assertion': 'off',
+    '@typescript-eslint/naming-convention': [
+      'error',
+      {
+        selector: 'interface',
+        format: ['PascalCase'],
+        custom: {
+          regex: '^I[A-Z]',
```

Comment:
What i really want is to lint the TypeScript interface names so they are always `ISomethingProps` but i can't seem to get this regex to work: `^I[A-Z]\w+Props\b`

Any ideas?

Reply:
Hi, I just checked the regex online and it looks like it's correct. Using https://regexr.com/ I tried with "iTechProps" and "ITechprops" and it wasn't a match -> correct as well.
![rege](https://user-images.githubusercontent.com/527144/93263863-039a4200-f7af-11ea-9676-3c5355463210.PNG)

Reply:
@daisygabi Thanks, i was using [https://regex101.com/](https://regex101.com/) and it was this worked `^I[A-Z]\w+Props\b` i wonder if it's prettier doing something weird

Reply:
Here ya go: `I([A-Z][a-zA-Z]+|)Props`

<img width="685" alt="Screenshot 2020-09-16 at 10 22 28" src="https://user-images.githubusercontent.com/4403089/93318383-87cbe400-f806-11ea-95c2-aa7088ca4a5f.png">

Reply:
Boogy time Thank you @drydenwilliams ... just for completeness: This is covers `State` too. e.g `IMyTestProps` | `IMyTestState`

```
I(([A-Z][a-z]+)+|)(Props|State)
```